xml/containers-buildah-overview.xml

```diff
@@ -0,0 +1,209 @@
+   <para>
+    Mount a working container's root filesystem for manipulation.
+   </para>
+  </listitem>
+  <listitem>
+   <para>
+    Use the updated contents of a container's root filesystem as a filesystem
+    layer to create a new image.
+   </para>
+  </listitem>
+  <listitem>
+   <para>
+    Delete a working container or an image and rename a local container.
+   </para>
+  </listitem>
+ </itemizedlist>
+ <para>
+  Compared to &deng;, &buildah; has several important advantages.
+ </para>
+ <itemizedlist>
+  <listitem>
+   <para>
+    &buildah; adheres the UNIX philosophy by focusing only on one task:
+    building container images.
+   </para>
+  </listitem>
+  <listitem>
+   <para>
+    The tool makes it possible to mount a working container's filesystem, so it
+    becomes accessible by host.
+   </para>
+  </listitem>
+  <listitem>
+   <para>
+    The process of building container images using &buildah; can be automated
+    using scripts.
+   </para>
+  </listitem>
+  <listitem>
+   <para>
+    Similar to &podman;, &buildah; supports the rootless mode.
+   </para>
+  </listitem>
+  <listitem>
+   <para>
+    It is possible to build images inside a container without mounting the
+    &docker; socket, which improves security.
+   </para>
+  </listitem>
+ </itemizedlist>
+ <sect1 xml:id="cha-podman-buildah">
+  <title>&podman; and &buildah;</title>
+
+  <para>
+   &podman; and &buildah; are complimentary tools for working with OCI images
+   and containers. While &buildah; focuses on building images, &podman; offers
+   commands and functions for maintaining and modifying images. It also makes
+   it possible to create, run, and maintain containers created from images.
+   Although &podman; allows you to build images via Dockerfile, it uses
+   &buildah;'s golang API to accomplish that.
+  </para>
+ </sect1>
+ <sect1 xml:id="cha-buildah-install">
+  <title>&buildah; Installation</title>
+
+  <para>
+   To install &buildah;, run the <command>sudo zypper in buildah</command>. Run
+   then <command>buildah --version</command> to check whether &buildah; has
+   been installed successfully.
+  </para>
+
+  <para>
+   If you already have &podman; installed and set up for use in the rootless
+   mode, &buildah; can be used in an unprivileged environment without any
+   further configuration. If you have to enable the rootless mode for
+   &buildah;, run the following command:
+  </para>
+
+<screen>&prompt.user;sudo usermod --add-subuids 200000-201000 --add-subgids
+   200000-201000 $USER</screen>
+
+  <para>
+   This command enables the rootless mode for the current user.
+  </para>
+
+  <note>
+   <title>&buildah; in the rootless mode</title>
+   <para>
+    In the rootless mode, &buildah; commands must be executed in the user
+    namespace of the user. To enter the user namespace, use the
+    <command>buildah unshare</command> command. Otherwise, the <command>buildah
+    mount</command> command will fail.
+   </para>
+  </note>
```

Comment:
I think we also need to have a Note here about giving access to the sle credentials when building as non-root.

Reply:
Could you elaborate on that? Thanks!

Reply:
When building a SLES based images with `builah` (or `podman build`) as a non root user. The credentials to access SCC need to be readable by that user. So basically what is mentioned here: https://github.com/SUSE/doc-sle/pull/665/files#diff-7a248a02092ab868664673831c4f2dd70f5a1e2a8e209178ec60066a8f73988dR100

I think this should be mentioned when talking about "rootless" container builds. (Though I am not sure where exactly ;-)  )